prediction/tools/model_warmup/README.md

```diff
@@ -0,0 +1,129 @@
+```
+
+Output
+
+```
+2020–01–23 23:58:38.999634: I tensorflow_serving/model_servers/server.cc:86] Building single TensorFlow model file config: model_name: resnet model_base_path: /models/resnet
+2020–01–23 23:58:39.000054: I tensorflow_serving/model_servers/server_core.cc:462] Adding/updating models.
+2020–01–23 23:58:39.000131: I tensorflow_serving/model_servers/server_core.cc:573] (Re-)adding model: resnet
+2020–01–23 23:58:39.135271: I tensorflow_serving/core/basic_manager.cc:739] Successfully reserved resources to load servable {name: resnet version: 1538687457}
+2020–01–23 23:58:39.135365: I tensorflow_serving/core/loader_harness.cc:66] Approving load for servable version {name: resnet version: 1538687457}
+2020–01–23 23:58:39.135419: I tensorflow_serving/core/loader_harness.cc:74] Loading servable version {name: resnet version: 1538687457}
+2020–01–23 23:58:39.136079: I external/org_tensorflow/tensorflow/cc/saved_model/reader.cc:31] Reading SavedModel from: /models/resnet/1538687457
+2020–01–23 23:58:39.151130: I external/org_tensorflow/tensorflow/cc/saved_model/reader.cc:54] Reading meta graph with tags { serve }
+2020–01–23 23:58:39.151230: I external/org_tensorflow/tensorflow/cc/saved_model/loader.cc:264] Reading SavedModel debug info (if present) from: /models/resnet/1538687457
+2020–01–23 23:58:39.152541: I external/org_tensorflow/tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
+2020–01–23 23:58:39.244208: I external/org_tensorflow/tensorflow/cc/saved_model/loader.cc:203] Restoring SavedModel bundle.
+2020–01–23 23:58:40.319685: I external/org_tensorflow/tensorflow/cc/saved_model/loader.cc:152] Running initialization op on SavedModel bundle at path: /models/resnet/1538687457
+2020–01–23 23:58:40.350975: I external/org_tensorflow/tensorflow/cc/saved_model/loader.cc:333] SavedModel load for tags { serve }; Status: success: OK. Took 1214904 microseconds.
+2020–01–23 23:58:40.355818: I tensorflow_serving/servables/tensorflow/saved_model_warmup.cc:105] No warmup data file found at /models/resnet/1538687457/assets.extra/tf_serving_warmup_requests
+2020–01–23 23:58:40.372651: I tensorflow_serving/core/loader_harness.cc:87] Successfully loaded servable version {name: resnet version: 1538687457}
+2020–01–23 23:58:40.375813: I tensorflow_serving/model_servers/server.cc:358] Running gRPC ModelServer at 0.0.0.0:8500 …
+[warn] getaddrinfo: address family for nodename not supported
+2020–01–23 23:58:40.377969: I tensorflow_serving/model_servers/server.cc:378] Exporting HTTP/REST API at:localhost:8501 …
+[evhttp_server.cc : 238] NET_LOG: Entering the event loop …
+```
+
+Please take a look at this message:
+
+```
+No warmup data file found at /models/resnet/1538687457/assets.extra/tf_serving_warmup_requests
+```
+
+I created [this](model_warmup.py) Python script to generate a warmup file which is relevant to this specific model and inference data.
```

Comment:
Microscopic nit: have the link text include the entire phrase "this Python script"